backend/src/guided-development.ts

```diff
@@ -70,25 +70,25 @@ export class GuidedDevelopment {
     if (itemAction) {
       if (context) {
         switch (itemAction.action.actionType) {
-          case ActionType.Command:
+          case "COMMAND":
```

Comment:
it gave you auto-complete suggestions here, correct?

Reply:
yes